app/controllers/reports_controller.rb

```diff
@@ -31,7 +37,10 @@ def report_params
     development_params = (1..ReportForm::DEVELOPMENT_OBJECTIVES).map do |n|
       "development_#{n}"
```

Comment:
But maybe you're right, I'll take a look. The problem with the objectives is that we also have to preserve the order (ie. if somebody fills 1. and then 5. objective). But maybe an array could clean the whole `ReportForm` object up.